mlflow/langchain/__init__.py

```diff
@@ -814,9 +822,10 @@ def _load_model_from_local_fs(local_model_path):
             config_path = None
 
         flavor_code_path = flavor_conf.get(_CODE_PATH, "chain.py")
+        flavor_model_code_config = flavor_conf.get(FLAVOR_CONFIG_MODEL_CODE)
```

Comment:
Forward comment: https://github.com/mlflow/mlflow/pull/11843#discussion_r1582536788

Reply:
Yes, removed in this PR https://github.com/mlflow/mlflow/pull/11843/files but need to port it over 😓